frontend/components/draft/lesson/draggable_block_list_client.tsx

```
@@ -1,10 +1,12 @@
 "use client";
-
 import { useCallback } from "react";
-import { Block } from "./add-block";
-import { AddBlock } from "./add-block";
+import AddLessonBlock, { Block } from "./add-tools";
```

ESLint

[error] 'AddLessonBlock' is defined but never used.

(@typescript-eslint/no-unused-vars)